modules/entrezdirect/esummary/main.nf

```diff
@@ -0,0 +1,39 @@
+process ENTREZDIRECT_ESUMMARY {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::entrez-direct=16.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/entrez-direct:16.2--he881be0_1':
+        'quay.io/biocontainers/entrez-direct:16.2--he881be0_1' }"
+
+    input:
+    tuple val(meta)
+    val uid
+    val database
+
+    output:
+    tuple val(meta), path("*.xml"), emit: xml_esummary
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def std_input = uid ? "-id ${uid}" : null
+    // std_input: single uid
+    if( std_input != null )
```

Comment:
I'm not sure what this is here exactly? Why would you want std_input when you can just supply with `-uid`? There is nothing to chain in teh command itself

Reply:
True, it is a line that I forgot to change in this commit, will remove it later.